pkg/mdformatter/linktransformer/link.go

```diff
@@ -201,14 +201,23 @@ func (v *validator) Close(ctx mdformatter.SourceContext) error {
 	})
 
 	merr := merrors.New()
+	base, err := os.Getwd()
+	if err != nil {
```

Comment:
Couple of things:

* `merr.Add` already checks if `err != nil` so no need to check again
* Don't wrap with `Wrapf` if there is no argument, use just `Wrap`
* No need to say something failed - you are wrapping failure after all, so just `resolve working dir` would be ok
* Are we sure we want to continue after those errors and not fail whole thing? why?

Reply:
Oh yes! Would be better to fail whole thing and return error rather than add to `merr`. Will implement this!